sources/views/activities.js

```diff
@@ -0,0 +1,86 @@
+import {JetView} from "webix-jet";
+import {activities} from "../models/activities";
+import {activitiesTypes} from "../models/activitiesTypes";
+import {contacts} from "../models/contacts";
+import activitiesForm from "./activitiesForm";
+
+export default class Activities extends JetView {
+	config() {
+		const button = {
+			css: "activities_button",
+			cols: [
+				{},
+				{
+					view: "button",
+					localId: "addButton",
+					value: "Add",
+					css: "webix_primary",
+					width: 150,
+					label: "Add activity",
+					type: "icon",
+					icon: "wxi-plus-circle",
+					click: () => this.showForm()
+				}
+			]
+		};
+
+		const table = {
+			view: "datatable",
+			editable: true,
+			select: true,
+			localId: "activitiesTable",
+			scroll: "auto",
+			columns: [
+				{id: "State", header: "", template: "{common.checkbox()}", checkValue: "Close", uncheckValue: "Open", width: 50},
+				{id: "TypeID", header: ["Activity type", {content: "selectFilter"}], collection: activitiesTypes, sort: "string"},
+				{id: "DueDate", header: ["Due Date", {content: "datepickerFilter"}], format: webix.i18n.longDateFormatStr, sort: "date", width: 150, editor: "date"},
+				{id: "Details", header: ["Details", {content: "textFilter"}], template: "#Details#", fillspace: true, sort: "string"},
+				{id: "ContactID", header: ["Contact", {content: "selectFilter"}], width: 150, options: contacts, sort: "string"},
+				{id: "edit", header: "", width: 50, template: "<span class='webix_icon wxi-pencil'></span>"},
+				{id: "delete", header: "", width: 50, template: "<span class='webix_icon wxi-trash'></span>"}
```

Comment:
From step 1 description
Required features:
on this step, buttons Delete and Edit are inactive